nemo/collections/asr/models/hybrid_rnnt_ctc_models.py

```diff
@@ -122,7 +122,6 @@ def transcribe(
             channel_selector (int | Iterable[int] | str): select a single channel or a subset of channels from multi-channel audio. If set to `'average'`, it performs averaging across channels. Disabled if set to `None`. Defaults to `None`. Uses zero-based indexing.
             augmentor: (DictConfig): Augment audio samples during transcription if augmentor is applied.
             verbose: (bool) whether to display tqdm progress bar
-            logprobs: (bool) whether to return ctc logits insted of hypotheses
```

Comment:
@titu1994 is this flag ever used? In metrics reworks it keeps popping up but i've never seen a need.

Reply:
It used to be used solely for ctc beam search, ice removed It from new transcribe API, now it's accessed via hypothesis object instead

Reply:
Okay then yay, purging flags.